src/helpers/converters.ts

```diff
@@ -39,6 +39,9 @@ import { erc20ABI } from "wagmi";
 import { formatBytes32String } from "./ethers";
 import { getQueryMetaData } from "./queryParsing";
 
+export const MIN_INT256 = -(BigInt(2) ** BigInt(255));
+export const MAX_INT256 = BigInt(2) ** BigInt(255) - BigInt(1);
+
```

Comment:
These constants can be imported from viem, see [here](https://github.com/wevm/viem/blob/main/src/constants/number.ts)